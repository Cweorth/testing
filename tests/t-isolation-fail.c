#include <cut.h>

TEST(1OK) {

}

TEST(2FAIL) {
    int *i = NULL;
    *i = 6;
}

TEST(3OK) {
    
}
