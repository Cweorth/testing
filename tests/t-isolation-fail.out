b'[  1] 1OK....................................................................\x1b[1;32mOK\x1b[0m\n[  2] 2FAIL................................................................\x1b[1;31mFAIL\x1b[0m\n    signal code: 11\n\n[  3] 3OK....................................................................\x1b[1;32mOK\x1b[0m\n\nSummary:\n  tests:       3\n  succeeded:   2\n  skipped:     0\n  failed:      1\n'